index.html

```diff
@@ -143,13 +143,15 @@ <h5>Priority L train subway Model</h5>
                         </a>
                 </div>
                 <div class="homebackground">
+                  <div class="container">
```

Comment:
I wrapped everything underneath `.homebackground` in a container, take a look at what bootstrap's css file defines for `.container` to see what happens when we do this